install.sh

```diff
@@ -19,13 +19,13 @@ installandconfigure(){
     sudo make install
 
     mkdir -p ~/.config/custard
-    cp ./examples/sxhkdrc ~/.config/custard/sxhkdrc
+    cp ./examples/sxhkdrc ~/.config/sxhkd/sxhkdrc
     cp ./examples/custard.sh ~/.config/custard/rc.sh
 
     cat <<EOF>> ~/.config/custard/start.sh
 \#!/bin/bash
 echo '' > /tmp/custard.log
-sxhkd -c ~/.config/custard/sxhkd &
+sxhkd -c ~/.config/sxhkd/sxhkd &
```

Comment:
Should be `~/.config/custard/sxhkdrc` (my fault, sorry)

Reply:
Could we please have this change added to `start.sh`? It's broken at the moment.

Reply:
@Snuggle As a temporary workaround, you can a) fix it yourself (just move the file to another path) or b) call `sxhkd` with the `-c` flag.